### DISCUSSION TITLE
ua-parser Compatibility with Python 3.11.x

### DISCUSSION THREAD

**Discussion by kmaheshwari-splunk** [ua-parser/uap-python#152] [General]
Hello folks,
The [Pypi documentation](https://pypi.org/project/ua-parser/) mentions that ua-parser v0.16.1 is compatible with Python >= 3.6.
Does this claim that the library is compatible with Python 3.11.x as well?
If not, is there a plan to release a Python 3.11.x compatible version?

Thanks!

**Comment by masklinn** [comment]
ua-parser should be compatible with 3.11, though I think the last time the CI ran was still on the alpha. UAP doesn't do anything really exciting at the python level though, so should be fine.

FWIW there's no ua-parser v6.0, not sure where you saw that (a fork?). Latest version is 0.16.1 as shown top-left of the page you link to.

**Reply by kmaheshwari-splunk** [reply]
Thanks for the information!
